org.mixedrealitytoolkit.uxcore/Editor/Inspectors/PressableButton/PressableButtonInspector.cs

```diff
@@ -210,6 +213,23 @@ private float DrawPlaneAndHandle(Vector3[] vertices, Vector2 halfExtents, float
         static bool advancedButtonFoldout = false;
         static bool editorFoldout = false;
 
+        protected override void DrawMRTKInteractableFlags()
```

Comment:
nit.   Add 

```cs
/// <inheritdoc />
```

To the top of the function, so the doc generation picks up the base description

Reply:
Done in latest commit.